src/csvcubed/models/sparqlresults.py

```diff
@@ -393,11 +395,66 @@ def _map_qube_component_sparql_result(
     return result
 
 
+def _get_column_components_and_check_for_cube_shape(
+    component: QubeComponentResult,
+    cube_shape: CubeShape,
+    csv_column_definitions: List[ColumnDefinition],
+    observed_value_columns: List[ColumnDefinition],
+):
+
+    if cube_shape == CubeShape.Standard and component.property_type == "Measure":
+        measure_column = first(
+            csv_column_definitions,
+            lambda c: c.property_url == "http://purl.org/linked-data/cube#measureType",
```

Comment:
Can we create a constant for this in a suitable central location and reference it here (and everywhere else it's used)?